.papr.yml

```diff
@@ -13,8 +13,7 @@ packages:
   - ansible
 
 tests:
-  - ansible-playbook -vi testnode, common/ans_ah_head-1_deploy.yml
-  - ansible-playbook -vi testnode, tests/improved-sanity-test/main.yml
+  - .test_director
```

Comment:
I think this needs to be `./.test_director`

See https://github.com/projectatomic/atomic/blob/master/.redhat-ci.yml#L19 as an example